addons/website_slides/models/slide_channel.py

```diff
@@ -468,14 +467,11 @@ def _compute_partner_has_new_content(self):
             new_slides = new_published_slides.filtered(lambda slide: slide.channel_id == channel)
             channel.partner_has_new_content = any(slide not in slide_partner_completed for slide in new_slides)
 
-    @api.depends('image_1920', 'channel_type')
-    def _compute_website_background_image_url(self):
-        for channel in self:
-            channel.website_background_image_url = (
-                channel.website_id.image_url(channel, 'image_256') if channel.image_1920
-                else '/website_slides/static/src/img/channel-%s-default.jpg'
-                     % ('training' if channel.channel_type == 'training' else 'documentation')
-            )
+    def _get_placeholder_filename(self, field):
+        image_fields = ['image_%s' % size for size in [1920, 1024, 512, 256, 128]]
+        if field in image_fields:
+            return 'website_slides/static/src/img/channel-%s-default.jpg' % ('training' if self.channel_type == 'training' else 'documentation')
```

Comment:
We could keep `website_background_image_url` and use it here
```suggestion
            return self.website_background_image_url
```
This way, it is immediately usable in `slide.slide`'s `_get_placeholder_filename` as it is now defined.
Otherwise (if we remove the field, and I agree it *is* cleaner, then you need an upgrade script as well), we'd have to port that logic in that `slide.slide` version to get the channel image as first fallback if defined (and not the placeholder of the channel image).